extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts

```diff
@@ -0,0 +1,518 @@
+
+	}
+
+	private createUserInputSection(): azdata.Component[] {
+		if (this._postgresModel.configLastUpdated) {
+			this.editWorkerNodeCount();
+			this.editCores(loc.coresRequest);
+			this.editCores(loc.coresLimit);
+			this.editMemory(loc.memoryRequest);
+			this.editMemory(loc.memoryLimit);
+		}
+
+		return [
+			this.createWorkerNodesSectionContainer(),
+			this.createCoresMemorySection(),
+			this.createConfigurationSectionContainer(loc.coresRequest, this.coresRequestBox!, '40px'),
+			this.createConfigurationSectionContainer(loc.coresLimit, this.coreLimitBox!, '40px'),
+			this.createConfigurationSectionContainer(loc.memoryRequest, this.memoryRequestBox!, '40px'),
+			this.createConfigurationSectionContainer(loc.memoryLimit, this.memoryLimitBox!, '20px')
+
+		];
+	}
+
+	private createWorkerNodesSectionContainer(): azdata.FlexContainer {
+		const inputFlex = { flex: '0 1 150px' };
+		const keyFlex = { flex: `0 1 250px` };
+
+		const flexContainer = this.modelView.modelBuilder.flexContainer().withLayout({
+			flexWrap: 'wrap',
+			alignItems: 'center'
+		}).component();
+
+		const keyComponent = this.modelView.modelBuilder.text().withProperties<azdata.TextComponentProperties>({
+			value: loc.workerNodeCount,
+			CSSStyles: { ...cssStyles.text, 'margin-block-start': '0px', 'margin-block-end': '0px' }
+		}).component();
+
+		const keyContainer = this.modelView.modelBuilder.flexContainer().withLayout({ alignItems: 'center' }).component();
+		keyContainer.addItem(keyComponent, { CSSStyles: { 'margin-right': '0px', 'margin-bottom': '15px' } });
+
+		const information = this.modelView.modelBuilder.button().withProperties<azdata.ButtonProperties>({
+			iconPath: IconPathHelper.information,
+			title: loc.workerNodesInformation,
+			width: '12px',
+			height: '12px',
+			enabled: false
+		}).component();
+
+		keyContainer.addItem(information, { CSSStyles: { 'margin-left': '5px', 'margin-bottom': '15px' } });
+		flexContainer.addItem(keyContainer, keyFlex);
+
+		const inputContainer = this.modelView.modelBuilder.flexContainer().withLayout({ alignItems: 'center' }).component();
+		inputContainer.addItem(this.workerBox!, { CSSStyles: { 'margin-bottom': '15px', 'min-width': '50px', 'max-width': '225px' } });
+
+		flexContainer.addItem(inputContainer, inputFlex);
+
+		return flexContainer;
+	}
+
+	private createConfigurationSectionContainer(key: string, input: azdata.Component, nestingLineHeight: string): azdata.FlexContainer {
+		const inputFlex = { flex: '0 1 150px' };
+		const keyFlex = { flex: `0 1 200px` };
+		const bottomLineFlex = { flex: `0 1 45px` };
+
+		const flexContainer = this.modelView.modelBuilder.flexContainer().withLayout({
+			flexWrap: 'nowrap',
+			alignItems: 'center'
+		}).component();
+
+		const leftLine = this.modelView.modelBuilder.divContainer().withProperties({
+			CSSStyles: { 'max-height': nestingLineHeight, 'min-height': nestingLineHeight, 'max-width': '1px', 'border-left-style': 'solid', 'border-left-color': '#ccc' }
```

Comment:
Did you test this on both light and dark themes? We don't really have theme-aware colors for extensions unfortunately, but in the meantime let's at least use a color that shows up nicely on both. The blue that we use for icons is typically a good choice.